src/app/machines/views/MachineDetails/MachineInstances/MachineInstances.tsx

```diff
@@ -111,6 +111,7 @@ const MachineInstances = (): JSX.Element => {
     <Row>
       <Col size={12}>
         <MainTable
+          aria-label="machine instances"
```

Comment:
Nice, appreciate you adding this and improving a11y/making this easier to test. 👍